### PR TITLE
Implement unpaired alignment 

### PR DESCRIPTION
**Pull request by Dohyun-s**
Dear sergey

Hope everything is going well with you. I have implemented unpaired alignment and fixed colabfold_search error. I have few things to note..

1. unserialize_msa in inputs.py seems to have bug in appending unpaired alignment.
2. One way of implmenting unpaired_alignment is run_mmseqs -> msa_to_str -> unserialize_msa -> get features. But it seems 0.1~0.2% difference in scores comparing when interaction_scan is not performed..
3. Another way I indented in code is run_mmseqs-> pass unpaired, paired alignment directly as function argument. The accuracy is same as when interaction_scan is not performed. (Couldn't find why those two methods differ in scores.)
4. Jax compilation fails in unpaired_alignment... Trying to debug this :)

Thank you 👍

### CODE REVIEW

**Comment by Dohyun-s** [comment]
Fixed 4. jax compilation error.. Need to make np.array to jnp.array...  (msa_cluster_size = jnp.array(feat["bert_mask"]).shape[0] )

**Comment by Dohyun-s** [comment]
Deleted previous commit using jnp.array. Get maximum of msa_cluster_size before run and pad the msas... Now jax compilation  is fixed but accuracy slightly differ 0.1%...

**Comment by Dohyun-s** [comment]
I have changed run condition in interaction_scan mode.. 

One thing to note is that if msa_size < 2556(2048+508), pop the sequences and run the model with recompiling... AlphaFold multimer set msa_feat size 508 and the extra_msa_feat 2048.. There were some cases where msa_size < 2556, the accuracy decreases... I couldn't solve the bug. So predicting with recompiled model for those cases will be safe way.
